### DISCUSSION TITLE
How do you delete transactions?

### DISCUSSION THREAD

**Discussion by bMorgan01** [powerpaul17/nc_money#70] [Q&A]
Hello! Thank you @powerpaul17 for your time and effort on this app.

I'm trying to get set up for the first time, but I've run into something. How do you delete transactions?

**Comment by powerpaul17** [comment]
You can delete transactions by deleting the splits. Currently there is no way to do it directly. Maybe it should be implemented, I'll put it on my todo list..

Also, at the moment the transactions persists in the list with a value of 0 until you switch accounts or reload. This is a known issue.
